Clean-code-cookbook-guide.html

```
@@ -0,0 +1,1110 @@
+            <li><span class="ref-num">11</span><span>Sandor Dargo's Blog – Tidy First? by Kent Beck（書評）<span class="ref-src"><a href="https://www.sandordargo.com/blog/2024/03/16/tidy-first-by-kent-beck" target="_blank" rel="noopener">https://www.sandordargo.com/blog/2024/03/16/tidy-first-by-kent-beck</a></span></span></li>
+            <li><span class="ref-num">12</span><span>thoughtbot – Sandi Metz' Rules For Developers<span class="ref-src"><a href="https://thoughtbot.com/blog/sandi-metz-rules-for-developers" target="_blank" rel="noopener">https://thoughtbot.com/blog/sandi-metz-rules-for-developers</a></span></span></li>
+            <li><span class="ref-num">13</span><span>InformIT – The Boy Scout Rule（Robert C. Martin, What Is Clean Code?より）<span class="ref-src"><a href="https://www.informit.com/articles/article.aspx?p=1235624&seqNum=6" target="_blank" rel="noopener">https://www.informit.com/articles/article.aspx?p=1235624&seqNum=6</a></span></span></li>
+            <li><span class="ref-num">14</span><span>Wikipedia – Design smell（Fowler・R.C. Martinの定義の整理）<span class="ref-src"><a href="https://en.wikipedia.org/wiki/Design_smell" target="_blank" rel="noopener">https://en.wikipedia.org/wiki/Design_smell</a></span></span></li>
+          </ol>
+        </div>
+
+        <div class="footer">本ガイドは2026年8月27日時点で参照可能な情報をもとに作成しています。</div>
+      </div>
+    </div>
+  </div>
+</div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)